galasa-parent/dev.galasa.framework.api.ras/src/test/java/dev/galasa/framework/api/ras/RequestorRasTest.java

```diff
@@ -0,0 +1,36 @@
+package dev.galasa.framework.api.ras;
+import java.util.HashMap;
+import java.util.Map;
+import dev.galasa.framework.api.ras.internal.ExtractQuerySort;
+
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.assertFalse;
+import org.junit.Test;
+
+public class RequestorRasTest {
+	
+	@Test
+	public void testWithParameterThatIsNull() {
+		Map<String, String[]> map = new HashMap<>();
+		
+		String[] stringArr = {"requestor:asc"};
+		
+		map.put("sort", stringArr);
+		
+		assertTrue(ExtractQuerySort.isAscending(map, "jim"));
+	}
+	
+	@Test
+	public void testWithParameterThatIsNotNull() {
+		
+		Map<String, String[]> map = new HashMap<>();
+		
+		String[] stringArr = {"requestor:desc"};
+		
+		map.put("sort", stringArr);
+		
+		assertFalse(ExtractQuerySort.isAscending(map, "requestor"));
+		
+	}
```

Comment:
:-).  you have proven 2 negatives here.
You need a junit that asserts true for requestor:asc.
Also junit tests for multiple sort options.
Then I will be happy